cfme/base/__init__.py

```diff
@@ -101,6 +113,12 @@ def slave_servers(self):
     def is_slave(self):
         return self in self.slave_servers
 
+    @property
+    def other_servers(self):
```

Comment:
This is a bit too ambiguous, how about `secondary_servers`

Reply:
Changed.